single-kernel/mol_dyn.cpp

```diff
@@ -132,8 +132,7 @@ class MolecularDynamicsBench
             j++;
         }
 
-        if(fabs(f.x() - output_acc[i].x()) > tolerance || fabs(f.y() - output_acc[i].y()) > tolerance ||
-            fabs(f.z() - output_acc[i].z()) > tolerance) {
+        if(sycl::distance(f, output_acc[i]) / sycl::length(f) > maxErr) {
```

Comment:
namespace `sycl::` here is undefined, as the kernel use `s::` for sycl namespace